### ISSUE TITLE
Auto-detect configuration files

### ISSUE THREAD

**Issue by ZanSara** [addisonElliott/pyqt5ac#34]
This is a feature request. If we can agree on a default name for the .yml configuration files, `pyqt5ac` could be modified to look for it at startup if no more specific instructions are given.

So, instead of typing something like
```
pyqt5ac --config pyqt5ac.yml
```
one could simply type
```
pyqt5ac
```
and, if `pyqt5ac.yml` exists in the folder, it would load it.

I can take on the implementation of this feature if you wish, but let's agree on the details first.

**Comment by zkovari** [comment]
Yes, I planned to support this. I think we can be a bit flexible with the naming and support more scenarios as well. Both `pyqt5ac.yml` and `.pyqt5ac.yml` seem reasonable for me. For json, we don't need to provide such support. Also, I think it's enough if only the CLI supports the auto-detection.

@ZanSara shall I assign to you?
